src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs

```diff
@@ -40,7 +41,8 @@ public BindAttribute(params string[] include)
         /// </param>
         public BindAttribute([NotNull] Type predicateProviderType)
         {
-            if (!typeof(IPropertyBindingPredicateProvider).IsAssignableFrom(predicateProviderType))
+            if (!typeof(IPropertyBindingPredicateProvider).GetTypeInfo()
+                    .IsAssignableFrom(predicateProviderType.GetTypeInfo()))
```

Comment:
My only issue with using that extension is that you end up with an unused `System.Reflection` using in DNX451. ifdefing that seems like more trouble than just calling `GetTypeInfo` twice over.